### ISSUE TITLE
[Bug]: Default designator disappears when selecting a notebook that isn't the default

### ISSUE THREAD

**Issue by catrobson** [opendatahub-io/odh-dashboard#1690]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Deploy type

Downstream version (eg. `RHODS 1.29`)

### Version

RHODS Sandbox version

### Current Behavior

When selecting a notebook that isn't the default notebook, the default designator on the screen disappears.  When selecting back to the default, it reappears.  

### Expected Behavior

~Default designator is always visible, regardless of radio selection of any notebook on the screen.~

UX decided on removing the `(default)` text (https://github.com/opendatahub-io/odh-dashboard/issues/1690#issuecomment-1692863646)

### Steps To Reproduce

1. In RHODS Sandbox version (sorry, cannot find version number), Launch Jupyter application.  
2. On Start Notebook Server page, it starts with Minimal Python 2023.1 (default) selected.
3. Selecting the following notebooks results in (default) suffix moving to that selection:  CUDA 2023.1, TensorFlow
2023.1, Standard Data Science 2023.1, PyTorch 2023.1
5. Selecting the following notebooks results in (default) suffix disappearing completely: oneAPI AI Analytics Toolkit (also why is this listed twice?), OpenVINO™ Toolkit v2022.1

Selected on initial default:
<img width="918" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/3739227/a4b6351b-d764-4382-b3e0-b6f218cb9383">

Selected on new item now with "default" suffix:
<img width="935" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/3739227/975931f2-67fb-402e-b6ea-caaadf5ef869">

Disappears completely:
<img width="880" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/3739227/88a5ae13-5b6c-4027-bfd2-2e3ed841ffd1">

### Workaround (if any)

Not blocking any user task, just confusing.

### What browsers are you seeing the problem on?

Chrome

**Comment by manaswinidas** [comment]
cc @xianli123 This is likely a UX issue. Default means default version and that's confusing.

**Comment by manaswinidas** [comment]
This does not happen in ds-projects

**Comment by xianli123** [comment]
@manaswinidas @catrobson Thanks for involving me. I often use the RHODS dashboard and I can't see the "Launch Jypter" page in the sandbox version. The below screenshot shows what I can see in the Sandbox version. I guess I may not have the permission to see that. 
<img width="1440" alt="Screenshot 2023-08-24 at 11 57 35" src="https://github.com/opendatahub-io/odh-dashboard/assets/53931519/39332d6b-1273-449f-b111-78d0562d425c">

Go back to this issue. I see the same interaction as what you shared @catrobson in the RHODS dashboard. I agree with you that the default designator should always appear regardless of whether the option is selected or not. 
As for the "default" designator, I think we can put it next to the subtitle rather than the main title as shown in the screenshot below. Because @manaswinidas mentioned that `Default means default version`, it'd better be next to the version info. What do you think?

> <img width="602" alt="Screenshot 2023-08-24 at 14 17 07" src="https://github.com/opendatahub-io/odh-dashboard/assets/53931519/ba206643-913c-4d60-b2ca-b51faeae1b7e">

**Comment by catrobson** [comment]
+1 that moving it next to the version number and not having it appear/disappear would be ideal if this is meant to designate that the user is currently selecting the default version.

**Comment by andrewballantyne** [comment]
Worth noting @xianli123 that `Minimal Python` is the name of the image. `2023.1` is the version (the versions expand will show this value and another -- n / n+1 concept). `Python v3.9` is the packaged version of the lang used in the image.

We have `(Recommended)` on DS Projects -- I think this is just the curse of the old UI. We can go with something small for now, but we will need to be really reworking this portion of the Dashboard to fixing the issue. I'll have to dig up the plans for that and prioritize it, it's getting a bit crazy having two UIs for this.

**Comment by xianli123** [comment]
Thanks for your explanation @andrewballantyne  
> 2023.1 is the version (the versions expand will show this value and another -- n / n+1 concept). 

Oh, sorry for misunderstanding the meaning of the version. 

> We can go with something small for now, but we will need to be really reworking this portion of the Dashboard to fixing the issue. I'll have to dig up the plans for that and prioritize it, it's getting a bit crazy having two UIs for this.

That makes sense. If you have any plans for this, please let me know and I can contribute to this part.

**Comment by xianli123** [comment]
@catrobson As [andrewballantyne](https://github.com/andrewballantyne) mentioned that `2023.1 is the version (the versions expand will show this value and another -- n / n+1 concept)` and I found that the **default version** is highlighted by the **Recommended**, so I think the smallest effective change is to remove all “Default" labels. And we can separate an issue to discuss how to improve the visibility of the "Recommended" version. Does it make sense?

> <img width="443" alt="Screenshot 2023-08-25 at 12 37 14" src="https://github.com/opendatahub-io/odh-dashboard/assets/53931519/8f7194f7-50c7-49ea-b387-4b2c835fa6ea">

**Comment by andrewballantyne** [comment]
Seems like an easy change to me... 👍

**Comment by andrewballantyne** [comment]
> Do we need to separate an issue to discuss how to improve the visibility of the "Recommended" version?

Yes please, log a new issue to state what you'd like changed. Worth noting, eventually (some distant future probably) we will be converting this design into the same design that is in the Workbench form for DS Projects. So perhaps not hugely needed imo, but I'll leave that up to you @xianli123

**Comment by xianli123** [comment]
@andrewballantyne Ok. I will dig into the new issue and try to come up with solutions.
